v2_bridges/int_on_evm/src/int/parse_tx_infos.rs

```diff
@@ -43,8 +44,15 @@ impl IntOnEvmEvmTxInfos {
             Self::get_supported_erc20_on_evm_logs_from_receipt(receipt, vault_address)
                 .iter()
                 .map(|log| {
+                    let erc777_mint_events = Erc777MintedEvents::try_from(receipt)?;
+                    // NOTE: There could be no such event, such as in the case of a h2h transfer
```

Comment:
That is something we need to address as the originating fields are useful for a h2h swap as well, e.g. staking PNT from BSC to Gnosis.

Reply:
Resolved in pvt, now awaiting review on updated router contract.